.github/workflows/watch-conda-forge.yml

```diff
@@ -0,0 +1,37 @@
+name: Check for new versions of pangeo-notebook metapackage on Conda Forge
+
+on:
+  schedule:
+    - cron: "0 * * * *"
```

Comment:
Seems like we'd be fine running this once daily: `0 0 * * *`

Reply:
I think hourly is going to be what we want. This decreases the likelihood that we have mismatched calver dates.

Reply:
I'll go ahead and merge as-is to try things out. I'm not too worried about mis-matched dates. I image we could have quite a few docker image tags between updates to the metapackages.